src/institutions/index.jsx

```diff
@@ -46,10 +46,7 @@ const _whatToRender = ({ filings, filingPeriod, institutions, submission }) => {
     const institution = institutions.institutions[key]
     const institutionFilings = filings[institution.id]
 
-    if (!institutionFilings || !institutionFilings.filing) {
-      // there are no filings
-      return <Institution key={i} institution={institution} />
-    } else if (!institutionFilings.fetched) {
+    if (!institutionFilings || !institutionFilings.fetched) {
```

Comment:
Doing this prevents a quick rendering of the "problem initializing your filing" error.